net/pfSense-pkg-pfBlockerNG-devel/files/usr/local/pkg/pfblockerng/pfblockerng.inc

```diff
@@ -319,15 +319,21 @@ function pfb_filter_service() {
 		/bin/ln "\${php_path}/php" "\${php_path}/php_pfb"
 	fi
 
-	# Start pfBlockerNG Firewall filter Daemon
-	if [ -e '/var/log/filter.log' ]; then
-		/usr/local/sbin/clog_pfb -f /var/log/filter.log | /usr/local/bin/php_pfb -f /usr/local/pkg/pfblockerng/pfblockerng.inc filterlog &
-	fi
+        # Start pfBlockerNG Firewall filter Daemon
+        file="/usr/local/sbin/clog"
```

Comment:
You don't need to declare a variable to use it just one time, please use the clog path on below conditional